packages/react/src/components/InAppMessaging/Backdrop/Backdrop.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { Flex } from '../../../primitives/Flex';
+
+import { BackdropProps } from './types';
+
+export default function Backdrop({
+  children,
+  ...rest
+}: BackdropProps): JSX.Element {
+  return (
+    <Flex className="amplify-backdrop" {...rest}>
```

Comment:
Hey @reesscot, the reason I did it this way was because
1. It is currently not a primitive, so I didn't want to add it to the primitives enum
2. I believe that this component has every potential to become a primitive (with much more robust features leveraging `backdrop-filter`)

It's a weird place for this to be in, admittedly, so given this rationale, does my proposed approach make more sense? I am open to suggestions on how to approach this as there certainly is not a precedence for doing this right now in the repo (i.e. Add a component scoped to a composed component with the *intent* of primitive-izing it later)

Reply:
Thinking we should hesitate towards adding anything new to `ComponentClassName` for the time being that is not a primitive. Currently, the `Authenticator` related classnames are not part of the `ComponentClassName` enum either, and we may want to keep separate enums for connected components and primitives